azure/template.json

```diff
@@ -0,0 +1,211 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "environmentName": {
```

Comment:
This requires two different values as resource naming differs from runtime environment name

Reply:
I've updated the template to use environmentFullName and resourceEnvironmentName parameters correctly.